modules/nf-core/kat/hist/main.nf

```diff
@@ -39,4 +48,23 @@ process KAT_HIST {
         kat: \$( kat hist --version | sed 's/kat //' )
     END_VERSIONS
     """
+
+    stub:
+    def args      = task.ext.args   ?: ''
+    def prefix    = task.ext.prefix ?: "${meta.id}"
+    def plot_type = args.contains("-p") || args.contains("--output_type") ?
+        args.split("(-p|--output_type) ")[-1].split(" ")[0] : 'png'
+    def hash_cmd  = args.contains("-d") || args.contains("--dump_hash") ? "touch ${prefix}-hash.jf27" : ""
```

Comment:
I think make the stub test simpler. The extra code is not necessary for anything

Reply:
@mahesh-panchal Shouldn't the files be generated only if the correct option is selected ?

Reply:
Given that you're adding the stub, and we're deprecating the module, it should not be used in a pipeline in any meaningful way. It's only for the nf-test and linting, and there it only needs to be the basic call.

Reply:
Okay, so do I even add a stub test ?

Reply:
You'll need it for linting right?

Reply:
Yes